src/daemon/utils_cache.c

```diff
@@ -203,6 +203,10 @@ static int uc_insert(const data_set_t *ds, const value_list_t *vl,
   ce->interval = vl->interval;
   ce->state = STATE_OKAY;
 
+  if (vl->meta != NULL) {
+    ce->meta = meta_data_clone(vl->meta);
```

Comment:
Hrm, I'm not sure what ce->meta is supposed to do but I think it was not meant for this purpose. It was introduced in 62e42f8dbbc99815f06d7204fc410bca9aece4ef but doesn't seem to be used since.

@octo can you comment on this?

Reply:
Friendly ping for @octo, do you recall history around this?